docs/antora.yml

```diff
@@ -0,0 +1,18 @@
+name: asciidoctor-reveal.js
+title: Asciidoctor reveal.js
+version: '4.0'
+nav:
+  - modules/setup/nav.adoc
+  - modules/converter/nav.adoc
+  - modules/ROOT/nav.adoc
+  - modules/project/nav.adoc
+asciidoc:
+  attributes:
+    uri-project-repo: 'https://github.com/asciidoctor/asciidoctor-reveal.js'
```

Comment:
Please use the url- prefix for URLs instead of uri-. URL is now the preferred over URI as specified by the What WG. See https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#URLs_and_URNs